django/contrib/admin/options.py

```diff
@@ -928,36 +928,32 @@ def get_paginator(
     ):
         return self.paginator(queryset, per_page, orphans, allow_empty_first_page)
 
-    def log_addition(self, request, obj, message):
+    def log_addition(self, request, queryset, message):
```

Comment:
We cannot change a signature.

Reply:
@felixxm  In the cases of current log_* methods, i.e. log_change and log_addition, should I replace them by new methods along with deprecation of current ones or keep them as they are and just call the log_actions method with single object?